lib/main.dart

```diff
@@ -1283,6 +1284,23 @@ class _MyHomePageState extends State<MyHomePage> {
                                           ? Colors.black26
                                           : Colors.black87,
                                     ),
+                                    FileIcon(
+                                      onPressed: (serialBusy ||
+                                              sections.sections.isEmpty)
+                                          ? null
+                                          : onExportWalls,
```

Comment:
Undefined function

Reply:
Fixed missing function in latest commit.